### ISSUE TITLE
Fix headings in the "Add facility" form

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8575]
## Area or Component

[FacilityForm](https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/Settings/Facility/FacilityForm.tsx#L331) under [these circumstances](https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/facilitySelect/WithFacility.tsx#L55)

## Quick Summary

The order of the headings `<h1>` and `<h2>` are not in order. The "Welcome to SimpleReport!" is marked as an `<h2>` but appears before the `<h1>` that is "Add facility"

### How to Replicate
1. Create a new org
2. Once created and verified, sign up as the new org admin user via the link you are emailed from Okta
3. After signing up successfully, you will be redirected to "Add facility"
4. That form contains the incorrect heading order

## Accessibility Violation
[heading-order](https://dequeuniversity.com/rules/axe/4.10/heading-order?application=AxeChrome)

## Action Requested

Adjust the heading order so that it only increases by one

## Screenshots (optional)

<img width="1326" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3NjQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b1d4837c46818392a5d384422fa5f58d7dbb653d/Screenshot%202025-03-04%20at%2012.54.54.png" alt="Screenshot 2025-03-04 at 12.54.54.png" />

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/Settings/Facility/FacilityForm.tsx#L331
